### PR TITLE
adds translator action

### PR DESCRIPTION
**Pull request by dvrajan**

### CODE REVIEW

File: statesman-engine/src/main/java/io/appform/statesman/engine/action/impl/TranslatorAction.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+            HandleBarsService handleBarsService,
+            @Named("eventPublisher") final EventPublisher publisher,
+            ObjectMapper mapper) {
+        super(publisher, mapper);
+        this.handleBarsService = handleBarsService;
+    }
+
+    @Override
+    protected JsonNode execute(TranslatorActionTemplate actionTemplate, Workflow workflow) {
+        String translator = actionTemplate.getTranslator();
```
Comment (tusharmndr) [inline statesman-engine/src/main/java/io/appform/statesman/engine/action/impl/TranslatorAction.java:37]: Use 
JsonNode workflowNode = mapper.valueToTree(workflow);
return toJsonNode(handleBarsService.transform(translator, workflowNode));

File: statesman-engine/src/main/java/io/appform/statesman/engine/handlebars/HandleBarsHelperRegistry.java [hunk 2]
```diff
@@ -135,9 +136,6 @@ private void registerGte() {
     private void registerGt() {
         handlebars.registerHelper("gt", (Helper<Number>) (aNumber, options) -> {
             val option = options.param(0);
-            if (option instanceof Long) {
```
Comment (tusharmndr) [inline statesman-engine/src/main/java/io/appform/statesman/engine/handlebars/HandleBarsHelperRegistry.java]: don't remove

File: statesman-engine/src/main/java/io/appform/statesman/engine/handlebars/HandleBarsHelperRegistry.java [hunk 3]
```diff
@@ -151,6 +149,17 @@ private void registerGt() {
         });
     }
 
+
+    private void registerAdd() {
+        handlebars.registerHelper("add", (Helper<Number>) (aNumber, options) -> {
+            val option = options.param(0);
+            if (option instanceof Integer) {
```
Comment (tusharmndr) [inline statesman-engine/src/main/java/io/appform/statesman/engine/handlebars/HandleBarsHelperRegistry.java]: handle all cases long, double,string

File: statesman-engine/src/main/java/io/appform/statesman/engine/action/impl/TranslatorAction.java [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+import lombok.extern.slf4j.Slf4j;
+
+import javax.inject.Inject;
+import javax.inject.Singleton;
+
+@Slf4j
+@Data
+@Singleton
+@ActionImplementation(name = "TRANSLATOR")
+public class TranslatorAction extends BaseAction<TranslatorActionTemplate> {
```
Comment (tusharmndr) [inline statesman-engine/src/main/java/io/appform/statesman/engine/action/impl/TranslatorAction.java:23]: Add in `MapBasedActionRegistry.class`

**Review by santanusinha — APPROVED** [review]

**Pull request merged**